olm-catalog/serverless-operator/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM alpine
+FROM centos
```

Comment:
If we can get this thing green, before merging, ideally we'd move this to ubi8 or whatever base image we're using for all our other operator images. For the purpose of hacking away, we can ignore it for now.

Reply:
I believe this is replaced in CI anyway by something else. I chose something that comes close to what CI will replace it with but the actual CI replacement has some RHEL dependencies which prevented `yum` to work locally.